tests/run_tests.sh

```diff
@@ -35,6 +35,12 @@ for PYTHON in /opt/python/*/bin/python; do
 		$PYTHON -c 'import sqlite3; print(sqlite3.sqlite_version); assert sqlite3.sqlite_version_info[0:2] >= (3, 34)'
 		# Make sure tkinter module can be loaded properly
 		$PYTHON -c 'import tkinter; print(tkinter.TkVersion); assert tkinter.TkVersion >= 8.6'
+		# Make sure openssl module can be loaded properly
+		if [ "${PYVERS}" == "3.6" ] || [ "${PYVERS}" == "3.7" ]; then
+			$PYTHON -c 'import ssl; print(ssl.OPENSSL_VERSION); assert ssl.OPENSSL_VERSION_INFO >= (1, 1, 1)'
+		else
+			$PYTHON -c 'import ssl; print(ssl.OPENSSL_VERSION); assert ssl.OPENSSL_VERSION_INFO >= (3, 0)'
+		fi
```

Comment:
Those checks will all fail:
system openssl is 1.0.2 on manylinux2014 for 3.6 & 3.7
system openssl is 1.1.1 on manylinux_2_28 so it shall be used for all python versions on that image.

Reply:
PS: the ssl-check.py a couple lines before makes sure that ssl can be imported and is working.
PS2: if you want to add a check on the version, it should be >= 1.1.1 for python >= 3.8 (I guess/hope that PyPy3.8+ is using something recent)

Reply:
I changed build-openssl and changed to change the other modules. Fixed.
This check is for version only.